examples_v20x/adc/adc.c

```diff
@@ -0,0 +1,80 @@
+#include "ch32fun.h"
+#include "ch32v20xhw.h"
+#include <stdio.h>
+
+
+void adc_init( void )
+{
```

Comment:
Please articulate how this is different from the regular ADC demo in the examples folder.